TourMate/TourMate/Frontend/Trip/Models/Trip.swift

```diff
@@ -18,6 +18,20 @@ struct Trip: CustomStringConvertible {
     let creationDate: Date
     var modificationDate: Date
 
+    // START date - END date
```

Comment:
This is the `startDate - endDate` formatting. Might as well shift the logic out of the view. It's the same for TripsView and TripView